src/GSMModem.cpp

```diff
@@ -45,3 +45,15 @@ String GSMModem::getIMEI()
 
   return imei;
 }
+
+String GSMModem::getICCID()
+{
+  String iccid;
+
+  iccid.reserve(20);
+
+  MODEM.send("AT+CCID");
+  MODEM.waitForResponse(120, &iccid);
```

Comment:
@ubidefeo I think we need to bump this timeout from 120, to something higher - maybe 1000?